modules/conductor/src/main/java/org/opencastproject/event/handler/OaiPmhUpdatedEventHandler.java

```diff
@@ -367,110 +123,59 @@ public void handleEvent(AssetManagerItem.TakeSnapshot snapshotItem) {
     }
 
     //An episode or its ACL has been updated. Construct the MediaPackage and publish it to OAI-PMH.
-    logger.debug("Handling {}", snapshotItem);
+    logger.info("Handling update event for media package {}",
```

Comment:
Useful at the moment to review this PR. Later, the debug level should go back to debug, though...

Reply:
Ok